aten/src/ATen/native/hammerblade/Uniform_.cpp

```diff
@@ -0,0 +1,32 @@
+#include <ATen/Dispatch.h>
+#include <ATen/hammerblade/HammerBladeContext.h>
+#include <ATen/HammerBladeGenerator.h>
+#include <ATen/native/hammerblade/Offload.h>
+#include <ATen/native/Distributions.h>
+
+namespace at { namespace native {
+
+Tensor hammerblade_common_seed_to_tensor_(uint32_t seed = 42) {
```

Comment:
There is another copy of this function in https://github.com/cornell-brg/hb-pytorch/blob/ef1429afc0ff523533520b23d25277c10b253fc1/aten/src/ATen/native/hammerblade/Distributions.cpp#L9-L23

Should we move it to a header and make it inline? (no need to change for this PR